src/README.md

```diff
@@ -25,7 +25,7 @@ Grafana Cloud: the LLM app plugin is installed for everyone, but LLM features ar
 OSS or Enterprise: install and configure this plugin with your OpenAI-compatible API key to enable various LLM-related functionality across Grafana.
 
 This includes new functionality inside Grafana itself, such as explaining panels, or in plugins,
-such as AI assistants for flamegraphs, error logs and more.
+such as AI assistants for flame graphs and error logs, automated incident summaries, and more.
```

Comment:
```suggestion
such as AI assistants for flame graphs, error logs, automated incident summaries, and more.
```

Reply:
Just a small nitpick.

Reply:
Ah, the AI assistants are only for flame graphs & logs... but clearly I need to make this sentence simpler :sweat_smile: